UPDATING.md

```diff
@@ -1190,7 +1190,7 @@ Examples:
 
 ### Semantics of next_ds/prev_ds changed for manually triggered runs
 
-next_ds/prev_ds now map to execution_date instead of the next/previous schedule-aligned execution date for DAGs triggered in the UI.
+next_ds/prev_ds and next_execution_date/prev_execution_date now map to execution_date instead of the next/previous schedule-aligned execution date for DAGs triggered in the UI.
```

Comment:
this is still a bit unclear.  didn't they always "map to execution_date"?
does it mean that they now sort by execution date without regard to triggering mechanism, where previously they only looked at scheduled runs?
or does it mean it now looks at _actual_ execution date instead of that which the schedule implies?

Reply:
Yeah maybe we need some better wording overall. @pbjc -> do you think you can do it or you need some help/additional explanation :)?